api-definitions/db_positions/api.yaml

```diff
@@ -0,0 +1,296 @@
+  PaymentNoticeNumber:
+    in: path
+    name: paymentnoticenumber
+    required: true
+    description:
+      The field ["Numero Avviso"](https://pagopa-specifichepagamenti.readthedocs.io/it/latest/_docs/Capitolo7.html#il-numero-avviso-e-larchivio-dei-pagamenti-in-attesa)
+      of pagoPa, needed to identify the payment. Format is `<aux digit (1n)>[<application code> (2n)]<codice IUV (15|17n)>`.
+      See [pagoPa specs](https://www.agid.gov.it/sites/default/files/repository_files/specifiche_attuative_pagamenti_1_3_1_0.pdf) for more info on this field and the IUV.
+    type: string
+    pattern: "^[0123][0-9]{17}$"
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  ApiKey:
+    type: apiKey
+    name: X-Functions-Key
+    in: header
+    description: The API key to access this function app.
+definitions:
+  Timestamp:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v10.2.3/openapi/definitions.yaml#/Timestamp"
+  FiscalCode:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v10.2.3/openapi/definitions.yaml#/FiscalCode"
+  OrganizationFiscalCode:
+    $ref: 'https://raw.githubusercontent.com/pagopa/io-functions/v0.64.0/api/definitions.yaml#/OrganizationFiscalCode'
+  PaymentNoticeNumber:
+    $ref: "https://raw.githubusercontent.com/teamdigitale/io-functions-commons/master/openapi/definitions.yaml#/PaymentNoticeNumber"
+  ProblemJson:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v18.0.7/openapi/definitions.yaml#/ProblemJson"
+  PaginationResponse:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v18.0.7/openapi/definitions.yaml#/PaginationResponse"
+  PaymentAmount:
+    description: |-
+      Amount of payment in euro cent. PagoPA accepts up to 9999999999 euro cents.
+    type: integer
+    minimum: 1
+    maximum: 9999999999
+    example: 15000
+  Iban:
+    type: string
+    pattern: '[a-zA-Z]{2}[0-9]{2}[a-zA-Z0-9]{1,30}'
+    example: IT60X0542811101000000123456
+  DetailResponse:
+    type: object
+    properties:
+      detail:
+        type: string
+    required:
+      - detail
+  DepartmentName:
+    type: string
+    description: |-
+      The department inside the organization that runs the service. Will
+      be added to the content of sent messages.
+    minLength: 1
+    example: ufficio tributi comune di Roma
+  Taxonomy:
+    type: string
+    pattern: "^[0-9]{3}-[a-zA-Z]$"
+    example: 010-TARI/TEFA
+    description: Payment Tax see [Tabella - Tassonomia dei servizi di incasso](https://docs.google.com/spreadsheets/d/13xOd__Qd4pwKHr3wjE-73NAB2O7UKmIt/edit#gid=1444196601)
+  Transfer:
+    type: object
+    properties:
+      partial_amount:
+        $ref: "#/definitions/PaymentAmount"
+      transfer_IBAN:
+        $ref: "#/definitions/Iban"
+      pa_fiscal_code:
+        $ref: "#/definitions/OrganizationFiscalCode"
+      reason:
```

Comment:
I'd try to be consistent in naming. you're using the prefix for iban field (transfer_iban) but not for the other fields (reason). I suggest to stick with one style (ie. remove the prefix everywhere when is not needed, and just use `iban`, it is implied that is the transfer iban since it's a field of the Transfer schema)